contracts/two-party-pol-holder/README.md

```diff
@@ -0,0 +1,62 @@
+# Two party POL holder
+
+## Responsibilities
+
+### Multiple parties
+
+Multiple parties are going to be participating, so the holder should store a list of whitelisted addresses.
```

Comment:
Two parties to be precise. This doesn't support an arbitrary number or parties yet.

Reply:
nice catch, will change